### DISCUSSION TITLE
Welcome to dactyl-lynx-keyboard Discussions!

### DISCUSSION THREAD

**Discussion by whitelynx** [whitelynx/dactyl-lynx-keyboard#4] [General]
Unedited template content courtesy of GitHub:

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Benson9954029** [comment]
Hello, I’m Benson. I just received the PCB and 3D printed parts.
I tried to fix the PCB with M.2 screws, but they don’t seem to fit properly.
Could you please tell me the recommended screw size for this PCB?
Thanks again for the project !

**Reply by whitelynx** [reply]
Sorry I didn't see your message earlier! I used M1.4*5 screws in mine, but I also didn't bother screwing in most of them; the keyswitches themselves hold the boards in pretty well. I just screwed in the ones at the corners of the thumb clusters.

**Reply by Benson9954029** [reply]
Thanks! I'll probably finish it in a few weeks.
I just finished soldering everything onto the PCBs.
